airbyte-integrations/connectors/source-shopify/source_shopify/shopify_graphql/bulk/query.py

```diff
@@ -2182,9 +2182,40 @@ class Product(ShopifyBulkQuery):
     query_name = "products"
     sort_key = "UPDATED_AT"
     # images property fields
-    images_fields: List[Field] = [Field(name="edges", fields=[Field(name="node", fields=["__typename", "id"])])]
+    images_fields: List[Field] = [
+        Field(name="edges", fields=[Field(name="node", fields=["__typename", "id", "url", "altText", "width", "height"])])
+    ]
     # variants property fields, we re-use the same field names as for the `images` property
-    variants_fields: List[Field] = images_fields
+    variants_fields: List[Field] = [
```

Comment:
There are 2 separate streams(`Product Images` and `Product Variants`) that already request most of the fields you have added in this pull request. Could you please clarify for my understanding why these streams can be used in your case? Does it make sense to you to update these to streams to sync fields you want?